server/web_ui/login_flows/pkg/README.md

```diff
@@ -0,0 +1,144 @@
+# Kanidm - Simple and Secure Identity Management
+
+<p align="center">
```

Comment:
Shouldn't this readme be specific to the component?

Reply:
It's really just a placeholder to make the build/package process less sad, and I think I can remove it anyhow

Reply:
Yep, gone.